kpet/run.py

```diff
@@ -18,26 +18,28 @@
 from kpet import utils
 
 
-def generate(args):
+def generate(template_content, kernel, output=None, arch='x86_64',
```

Comment:
Niice, I like how you're spelling out the parameters.

I think it's better to not have any defaults for things like the architecture, and no defaults at all unless we use those defaults. Otherwise it's easy to make a mistake and just forget to specify the argument, where it was actually needed. We already have defaults at the command-line level and that's all that's required and being used right now.

Do you think you can make all the parameters required?

Reply:
Done